src/v/cluster/cloud_storage_size_reducer.h

```diff
@@ -54,7 +54,7 @@ class cloud_storage_size_reducer {
       ss::sharded<partition_leaders_table>& leaders_table,
       ss::sharded<rpc::connection_cache>& conn_cache,
       size_t batch_size = topic_table_partition_generator::default_batch_size,
-      uint8_t retries_allowed = 3);
+      uint8_t retries_allowed = default_retries_allowed);
```

Comment:
> some minor changes

even for trivial changes we try to write a meaningful commit message. for example,

```
cluster: use named constant
```
and
```
cluster: remove redundant `virtual` keyword
```

Reply:
commit message amended.